### PR TITLE
2015 rp update

### PR DESCRIPTION
**Pull request by zmmille2**
Changed around emails to properly refer to 2015 dates, changed where formsites point, added link to new sponsorship pdf.

### CODE REVIEW

File: liquid/templates/conference/emails/jobfair_invite.txt [hunk 1]
```diff
@@ -1,31 +1,36 @@
...
+We're excited to extend an invitation to {{ company.company_name }} to attend this year's annual Reflections | Projections job fair. The job fair will be held on Friday, October 2nd, 2015. Reflections | Projections is one of the largest student run conferences in the US, with over 800 students coming every year to the job fair, and we'd love to have {{ company.company_name }} involved.
 
-The job fair will be held on the Friday of the conference weekend, and serves as a kickoff event for the conference. The price for a job fair booth is $1250.
+We've revamped what sponsorship will look like this year, with three separate tiers to get involved with. We're offering companies more than ever before, and there are some fantastic new ways to get more out of Reflections | Projections. To learn more about our tiered sponsorship, see the following pdf (acm.uiuc.edu/static/sponsorship.pdf), which breaks down the tiers into more specific detail.
```
Comment (colegleason) [inline liquid/templates/conference/emails/jobfair_invite.txt]: you probably should add http:// so email clients turn it into a link.

Comment (CJS7070) [inline liquid/templates/conference/emails/jobfair_invite.txt]: Also, please use illinois.edu instead of uiuc.edu.

Comment (zmmille2) [inline liquid/templates/conference/emails/jobfair_invite.txt]: next commit fixes both of these issues!

**Comment by colegleason** [comment]
LGTM

**Pull request merged**